### DISCUSSION TITLE
账号被封禁了，十分难过

### DISCUSSION THREAD

**Discussion by lgbt8887** [BurnHair/docs#153] [技术支持]
大佬，我诚恳的请求解封，我就 大号的用的的7年git注册的，今天莫名被封了，我不知道怎么肥事 ，最近就公司和家中来回这两个地方交替使用，里面还有一点刀，正准备这几天 充值一下，15刀，有点大，麻烦开几个小额度，方便不，十分感谢了
![image](https://github.com/BurnHair/docs/assets/170690713/2a5af59e-f885-458d-be90-020c86105261)
 "error": {
    "message": "用户5906已被封禁： (request id: 20240708135601536421592hNEjrw1s)",
    "type": "one_api_error"
  }

**Comment by BennyThink** [comment]
好了

**Reply by lydong0925** [reply]
大佬5906是触发什么封控了嘛，咋肥事，又被疯啦

**Reply by BennyThink** [reply]
你发帖的账户 是 8063，这里又说 5906。 邀请记录来看你也自己邀请自己。

**Reply by lydong0925** [reply]
大佬打扰了，之前可能注册的手机一直在线，被同一个 ip. 封控锁定了😨，以后就用lydong0925..这一个号啦，别的号我就不用了，严格执行平台规范，want大佬解封，辛苦了，一会去充值一波，表示支持😅😅😅

因为昨天没有办法登陆冒火，所以进不了。申诉这个界面。

**Reply by lydong0925** [reply]
8887的这个号登录的git.发的帖。这个是新号，我就不用了。以儆效尤

**Reply by BennyThink** [reply]
1. 这里是GitHub，你能不能在这里反馈问题，和我的封禁无关
2. 可以解封 lydong0925，但是我会清零所有额度。因为我无法判断真实额度是多少

**Reply by lydong0925** [reply]
好的 ，我知道拉，git是我使用上的问题，对您造成的困扰我十分抱歉，欢迎清零，我重新开始，重新做人

**Reply by BennyThink** [reply]
可以了。

**Reply by inoiglias99** [reply]
Hello admin. I am a free user. I do daily check in in burn.hair. Will your service discontinued?  I saw a message in the site where I could not check in for today. Can you clarify please?
Thanks.

**Reply by BennyThink** [reply]
1. Your account status is normal so you should be able to check in.

2. The service for paid users will continue.

3. We reserve the right to claim back free quota because we will only focus on paid users. See your token page and ToS to learn more
